addons/mail/static/src/models/rtc_call_viewer.js

```diff
@@ -34,6 +34,19 @@ registerModel({
         onLayoutSettingsDialogClosed(ev) {
             this.toggleLayoutMenu();
         },
+        /**
+         * @param {MouseEvent} ev
+         */
+        onMouseLeave(ev) {
```

Comment:
`onMouseLeave` => `onMouseleave` (the event is named `mouseleave`, thus no capital letter on `L`)